### DISCUSSION TITLE
Startswith和Endswith的剩余部分

### DISCUSSION THREAD

**Discussion by zyckk4** [nonebot/discussions#517] [想法]
在按开头（或结尾）匹配触发后，用户可能更关心开头（或结尾）外的剩余部分。nonebot.params能否考虑提供相应注入？

**Comment by yanyongyu** [comment]
这个其实可以直接通过removeprefix和removesuffix得到

**Reply by zyckk4** [reply]
主要是还得event.message获取原消息，假如有相应注入，可能就不用带event了
